Marlin/src/lcd/e3v2/proui/meshviewer.cpp

```diff
@@ -92,13 +92,13 @@ void MeshViewer::drawMeshPoint(const uint8_t x, const uint8_t y, const float z)
         DWINUI::drawSignedFloat(meshfont, 1, 1, px(x) - 3*fs, py(y) - fs, z);
         break;
       case -99 ... -1:
-        sprintf_P(str_1, PSTR("-.%02i"), -v);
+        sprintf_P(str_1, PSTR("%.2f"), z);
```

Comment:
Sorry! `printf` on these platforms doesn't support the `f` formatter. Use `MString` with `p_float_t`.